Todo.WillowBear/Todo.WillowBear/Data/TodoDbContext.cs

```diff
@@ -0,0 +1,160 @@
+using Microsoft.EntityFrameworkCore;
+using Todo.WillowBear.Models;
+using Serilog;
+
+namespace Todo.WillowBear.Data
+{
+    public class TodoDbContext : DbContext
+    {
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            optionsBuilder.UseSqlite("Data Source=TodoList.db");
+        }
+
+        // 
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.ApplyConfiguration(new TodoItemConfiguration());
+        }
+        public DbSet<TodoItem> TodoItems { get; set; }
+
+        //Add Todo
+        /// <summary>
+        /// Adds param to Database table
+        /// </summary>
+        /// <param name="todoItem"></param>
+        public void AddTodoItem(TodoItem todoItem)
```

Comment:
Please remove all methods that aren't being used 😁

Reply:
Fixed.